k-distribution/tests/regression-new/map-symbolic-tests-haskell/inkeys

```diff
@@ -0,0 +1,9 @@
+claim <k> inkeys ( X:MyId in_keys ( .Map  ) ) => . </k>
+claim <k> inkeys ( Y:MyId in_keys ( M:Map ) ) => . </k> requires Y in_keys(M)
+claim <k> inkeys ( Y:MyId in_keys ( M:Map ) ) => . </k> requires notBool Y in_keys(M)
```

Comment:
These tests are missing from the Java backend test suite, even though they exist in the `inkeys` file.

Reply:
We don't expect these to pass until we finish kframework/kore#2249.